src/cloud-api-adaptor/cmd/process-user-data/main.go

```diff
@@ -19,7 +19,7 @@ const (
 	providerAzure = "azure"
 	providerAws   = "aws"
 
-	defaultAgentConfigPath = "/etc/agent-config.toml"
+	defaultAgentConfigPath = "/run/peerpod/agent-config.toml"
```

Comment:
the defaultAgentConfigPath is changed here but
`src/cloud-api-adaptor/podvm/files/etc/systemd/system/kata-agent.service` didn't use the new changed path.

Reply:
@liudalibj thanks for finding it, I think that's another reason why it works on ubuntu but not on fedora, fedora altered the path to `/run/peerpod/agent-config.toml` here https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/podvm-mkosi/mkosi.skeleton/usr/lib/systemd/system/kata-agent.service.d/10-override.conf

Reply:
I guess we should change to use `/run/peerpod/agent-config.toml` on ubuntu also.

Reply:
I'll give a try after update the path...

Reply:
Agree, so for ubuntu this file need be updated in this pr, 

https://github.com/confidential-containers/cloud-api-adaptor/pull/1853/files#diff-7a67c51609d012acb98c931da3f33e5494e6d5722e99cd4d9e294ebcaf15b8faL10